### PR TITLE
Sort by size mixing files and folders

### PR DESCRIPTION
**Pull request by abelgardep**
Implements #2717

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java [hunk 1]
```diff
@@ -242,14 +242,12 @@ public static RemoteFile fillRemoteFile(OCFile ocFile) {
...
 
-        Collections.sort(files, new Comparator<OCFile>() {
-            public int compare(OCFile o1, OCFile o2) {
-                if (o1.getModificationTimestamp() == 0 || o2.getModificationTimestamp() == 0) {
-                    return 0;
-                } else {
-                    Long obj1 = o1.getModificationTimestamp();
-                    return val * obj1.compareTo(o2.getModificationTimestamp());
-                }
+        Collections.sort(files, (ocFile1, ocFile2) -> {
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java]: You are not using `import java.util.Comparator;` anymore, are you?

**Comment by davigonz** [comment]
@abelgardep just a very tiny change

File: owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java (hunk 1 above)
Comment (abelgardep) [reply owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java]: Removing it..

**Comment by davigonz** [comment]
@jesmrec code approved, ready to test

**Comment by jesmrec** [comment]
Checked in the same way as #2710. 

Tested with

Xiaomi MiA2 - Android 9
Nexus 9 - Android 7

Approved

**Comment by davigonz** [comment]
Tests are stuck, I will close and reopen the PR to relaunch them

**Pull request merged**